tests/lib/bin/retry

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+COMMAND="${1:-true}"
+MAX_ATTEMPTS="${2:-20}"
+SLEEP_TIME="${3:-.5}"
+
+for _ in $(seq "$MAX_ATTEMPTS"); do
+    if $COMMAND; then
```

Comment:
I think it would be best to use `shift` the command line options away and finally execute "$cmd" "$@"

Reply:
done